### PR TITLE
Enable openshift/hypershift-toolkit repository

### PR DESCRIPTION
**Pull request by csrwng**
Initial prow config for the openshift/hypershift-toolkit repository

### CODE REVIEW

File: ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-master.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+tag_specification:
+  name: '4.3'
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-master.yaml]: Should this be 4.4?

File: ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-master.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  namespace: ocp
+base_images:
+  base:
+    name: '4.3'
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-master.yaml]: Should this be 4.4?

File: ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-master.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-master.yaml]: Should this be 1.13?

File: core-services/prow/02_config/_plugins.yaml [hunk 4]
```diff
@@ -1802,6 +1802,8 @@ plugins:
   - approve
```
Comment (stevekuznetsov) [inline core-services/prow/02_config/_plugins.yaml]: You didn't add this to any automatic merges. Did you use `make new-repo`?

Comment (csrwng) [reply core-services/prow/02_config/_plugins.yaml]: No, indeed.... this is new sorcery I was not aware of! Cool

**Comment by csrwng** [comment]
Re-did based on `make new-repo`. Pls take a look @stevekuznetsov

**Pull request merged**